com.unity.render-pipelines.universal/Runtime/Passes/CopyDepthPass.cs

```diff
@@ -129,7 +129,12 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
                     // scaleBias.y = scale
                     // scaleBias.z = bias
                     // scaleBias.w = unused
-                    float flipSign = (cameraData.IsCameraProjectionMatrixFlipped()) ? -1.0f : 1.0f;
+                    bool renderingToBackBufferTarget = ScriptableRenderer.current.cameraColorTarget == BuiltinRenderTextureType.CameraTarget;
+                    bool renderingToTexture = !renderingToBackBufferTarget || cameraData.camera.targetTexture != null;
+                    // In game view final target acts as back buffer were target is not flipped
+                    bool isGameViewFinalTarget = (cameraData.cameraType == CameraType.Game && destination == RenderTargetHandle.CameraTarget);
+                    bool yflip = (renderingToTexture && !isGameViewFinalTarget) && SystemInfo.graphicsUVStartsAtTop;
+                    float flipSign = yflip ? -1.0f : 1.0f;
```

Comment:
I am not super happy about this, open for suggestions

Reply:
I'm worried as well a little bit about this one. It would be better if we would fix the cameraData.IsCameraProjectionMatrixFlipped() instead.

Reply:
You have `bool yflip = (renderingToTexture && !isGameViewFinalTarget) && SystemInfo.graphicsUVStartsAtTop;`

which is same as `(renderingToTexture && SystemInfo.graphicsUVStartsAtTop) && !isGameViewFinalTarget`

and `renderingToTexture && SystemInfo.GraphicsUVStartsAtTop` is same as `IsCameraProjectMatrixFlipped`

so it could be rewritten as `bool yflip = cameraData.IsCameraProjectionMatrixFlipped() && !isGameViewRenderTarget` ?

Reply:
Fixed